addon/components/frost-sidebar.js

```diff
@@ -17,8 +19,20 @@ export default Component.extend(PropTypesMixin, {
       PropTypes.object,
       PropTypes.EmberObject
     ]).isRequired,
+    label: PropTypes.oneOfType([
+      PropTypes.EmberComponent,
+      PropTypes.string
+    ]),
 
     // Actions
     onToggle: PropTypes.func.isRequired
+  },
+
+  // == Computed Properties ===================================================
+
+  @readOnly
+  @computed('label')
+  isLabelComponent (label) {
+    return validators.EmberComponent(null, 'label', label, null, false, false)
```

Comment:
Overall I am happy with this PR and it looks good, but I have one question about this part.  I understand what you are doing and why - I have no problem with that.   Can we accomplish the same thing though by using:

`return typeOf label !== 'string' // where typeOf was destructured off of Ember`

?  This way we do not have to import `{validators}` nor have to wrangle with its list of arguments, reason about how to use it, etc.  The other is much simpler and accomplishes the same thing (unless I've missed something)

Reply:
I guess that does work since we're only expecting `label` to be either a string or a component. I'll make the change